polyfill/test/validStrings.mjs

```diff
@@ -174,7 +205,9 @@ function seq(...productions) {
 // Grammar productions, based on the grammar in RFC 3339
 
 // characters
-const sign = character('+-−');
```

Comment:
This change isn't correct - Unicode minus signs should still be accepted everywhere ASCII minus signs are accepted. It's just that this PR should preserve the current observable behaviour of rejecting `-000000` but allowing `\u2212000000`. I think if you just revert this hunk, the PR will be correct.

Reply:
Ah, I see -- OK, fixed